src/MCPServer/share/mysql_dev_9647_archivesspace_inherit_notes.sql

```diff
@@ -0,0 +1,5 @@
+-- Add column inherit_notes column to administration_archivesspaceconfig
+ALTER TABLE `administration_archivesspaceconfig` ADD COLUMN `inherit_notes` bool NOT NULL AFTER `repository`;
```

Comment:
syncdb (not migrate) will only create tables, not modify them.  On a fresh install, this will work correctly, but on an upgrade the new column won't be created.  SQL changes or a real migration will fix this. Since we expected migrations to make it into 1.5, and this is scheduled for 1.6, I think we can handle this when Django migrations come in.